src/js/tech/html5.js

```diff
@@ -70,6 +71,24 @@ class Html5 extends Tech {
       }
     }
 
+    let trackTypes = ['audio', 'video'];
+
+    // ProxyNativeTextTracks
+    trackTypes.forEach((type) => {
+      let capitalType = toTitleCase(type);
+
+      if (!this[`featuresNative${capitalType}Tracks`]) {
+        return;
+      }
+      let tl = this.el()[`${type}Tracks`];
+
+      if (tl && tl.addEventListener) {
+        tl.addEventListener('change', Fn.bind(this, this[`handle${capitalType}TrackChange_`]));
```

Comment:
Follow-up: it's better/safer to use `this.on(trackList, 'event', listener)` __but__  the current implementation of on() only works with Components or DOM nodes. I'm going to file an issue to fix that so we don't have to worry about manually cleaning up listeners for any EventTarget-like object.  We can stick with this for now.